api/python/cellxgene_census/src/cellxgene_census/experimental/ml/pytorch.py

```diff
@@ -505,7 +510,7 @@ def _subset_ids_to_partition(
 
         # subset to a single partition
         # typing does not reflect that is actually a List of 2D NDArrays
-        partitions: List[npt.NDArray[np.int64]] = np.array_split(np.array(ids_chunked, dtype=object), num_partitions)
+        partitions: List[npt.NDArray[np.int64]] = np.array_split(np.array(ids_chunked), num_partitions)
```

Comment:
was causing a tiledbsoma assertion to fail, and not even sure why this was originally typed as `object`